src/style.css

```diff
@@ -0,0 +1,147 @@
+body {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  background: #e5e5e5;
+}
+
+:root {
+  --fonovy-color: #fff;
+  --button-color: #00acdc;
+  --secondary_text-color: #616070;
+  --main_text-color: #060b35;
+}
+
+.conteiner {
+  width: 200px;
+  height: 408px;
```

Comment:
you must not to fix height and width in container

Reply:
![image](https://github.com/mate-academy/layout_product-cards/assets/55200530/1c966d65-40a2-4d87-a354-b8a4724a8bc1)
still not resolved, why you've set fixed size for container? 
![image](https://github.com/mate-academy/layout_product-cards/assets/55200530/4c8e526f-1ad0-4abf-b258-a46b1571ade3)